app/services/api/v1x0/catalog/service_plan_reset.rb

```diff
@@ -25,7 +26,7 @@ def process
         private
 
         def reimport_from_topo
-          Catalog::ImportServicePlans.new(@plan.portfolio_item_id, :force_reset => true).process
+          @reimported_service_plan = Catalog::ImportServicePlans.new(@plan.portfolio_item_id, :force_reset => true).process.json
```

Comment:
This should probably be moved to a v1x1 service_plan_reset service class right? Since we're keeping the old functionality for v1.0.

~That way we wouldn't have to change the v1.0 spec as well.~
EDIT: after looking through that spec I wasn't validating the contents in the beginning, so that spec change is great!